app/assets/scripts/components/line-chart.js

```diff
@@ -20,31 +20,53 @@ export default function LineChart({ data, xUnit }) {
     legend: {
       display: false,
     },
+    tooltips: {
+      intersect: false,
+    },
     scales: {
       yAxes: [
         {
           ticks: {
             beginAtZero: true,
             maxTicksLimit: 5,
+            callback: function (value, index) {
+              return index === 0 && yUnit ? [value, `(${yUnit})`] : value;
+            },
+            fontSize: 14,
           },
           gridLines: {
             drawOnChartArea: false,
           },
+          scaleLabel: {
+            display: !!yLabel,
+            labelString: yLabel,
+            fontSize: 14,
+          },
         },
       ],
 
       xAxes: [
         {
           type: 'time',
           time: {
+            tooltipFormat: 'MMM D, YYYY',
             unit: xUnit || 'day',
+            displayFormats: {
+              day: 'MMM D, YYYY',
```

Comment:
Perhaps this should be formatted as MM/DD/YY to save space? current formatting seems crowded and repetetive.